src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -2,10 +2,8 @@
 
 import java.util.List;
 
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+public interface MachineProducer<M> {
```

Comment:
Should this Producer be able to accept a `String` or `Integer` as a type?

Reply:
No, It shouldn't)